### ISSUE TITLE
[accessibility] Consider replacing `aria-pressed` with `aria-expanded` on the attribution control

### ISSUE THREAD

**Issue by Malvoz** [mapbox/mapbox-gl-js#11034]
Regarding:

https://github.com/mapbox/mapbox-gl-js/blob/36533f3683dd48837ab07095fb6484165d9ee400/src/ui/control/attribution_control.js#L103-L111

[`aria-pressed`](https://www.w3.org/TR/wai-aria/#aria-pressed) is similar to [`aria-expanded`](https://www.w3.org/TR/wai-aria/#aria-expandedd) in that it indicates the state of the button, but `aria-expanded` is more appropriate as it specifically:

> Indicates whether the element, or another grouping element it controls, is currently expanded or collapsed.

The distinction is that with `aria-expanded` the (non-visual) user will expect other ("new") content to be available after pressing the button. This is mostly important as there is no focus delegation (which is fine if it's conveyed to the user that new content is available through `aria-expanded`).
